### DISCUSSION TITLE
Loosing synch at same time every day [BUG]

### DISCUSSION THREAD

**Discussion by KaAProjekt** [Chia-Network/chia-blockchain#5315] [Support]
**Describe the bug**
It seems like my internetprovider is resetting my connection every night between 3:50am and 4am.
I will loose synch after that (which seems okay), yet after being connected again, node stays "not synched" for 5 and more hours.

When I close the node completely and restart it, it eventually gets synched and is fully online again after 10-15 Minutes.

I have had restarts of the router during the day without any issues related to synching, or after shortly loosing synch being reconnected by the node itself. But it does not recover itself at that specific time.

I mean, why can node recover a regular disconnection/reset but not that one during the night? 
Is there any way to get the node to correct this issue? Even just pressing reload on node doesn't do anything. Only thing that works is the shutdown and restart of node.
It's a giant pain in the butt to loose synch in the midle of the night and needing to manually restart node every night.

This issue seems to occur only with the version 1.1.4/1.1.5 ; before I had some disconnects and not being synched, but node recovered itself all the time.

**To Reproduce**

- OS: [Win 10 Pro 64]
Using GUI for windows

**Comment by boombel666** [comment]
i have the same problem. does anyone know how to fix it ?

**Comment by Tom29a** [comment]
Maybe a fix ip adress can help

**Comment by KaAProjekt** [comment]
> 
> 
> Maybe a fix ip adress can help

Was one of my first thoughts as well, yet doesn't seem to work...

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
